### ISSUE TITLE
Upgarding boot2docker does not upgrade docker version v1.4.1

### ISSUE THREAD

**Issue by kameshsampath** [boot2docker/boot2docker#721]
I just did upgrade boot2docker to v1.4.1 from my existing installation of Mac, after upgrade when i did docker  -v I still see it as `1.3.2` but when I checked the boot2docker github repo for 1.4.1 they say the Docker version to be 1.4.1

Is there anything else required to upgrade docker version as well ?

-Kamesh

**Comment by walterdolce** [comment]
@kameshsampath are you running `docker -v` from within the VM?

Assuming you already have the VM, if you do:
```
> boot2docker start
> boot2docker ssh
> docker -v
```

Do you still get `1.3.2` or `Docker version 1.4.1, build ...` ?

**Comment by walterdolce** [comment]
Sorry. I might have misread your message. You say: "I just did upgrade boot2docker to v1.4.1".
Does that mean that you ran `boot2docker upgrade` ?

If so, you might consider reinstalling the latest package available and try again.

**Comment by kameshsampath** [comment]
yes I did boot2docker upgrade and then tried docker -v which is still showing 1.3.2, i can understand that doing reinstall will solve the issue but then ideally upgrade should :)

**Comment by walterdolce** [comment]
I'd agree with you, yes. I'll let the others continue from this point on then :)

**Comment by nathanleclaire** [comment]
@kameshsampath If the boot2docker binary version is 1.4.1, you can run `boot2docker upgrade` to get the latest everything (b2d binary, docker client binary, and ISO).

**Comment by foresmac** [comment]
I'm having a similar problem:
``` bash
(master) Violet:trunq-api > boot2docker upgrade
Latest release for boot2docker/boot2docker is v1.5.0
Downloading boot2docker ISO image...
Success: downloaded https://github.com/boot2docker/boot2docker/releases/download/v1.5.0/boot2docker.iso
	to /Users/foresmac/.boot2docker/boot2docker.iso
(master) Violet:trunq-api > boot2docker up
Waiting for VM and Docker daemon to start...
...............ooo
Started.
Writing /Users/foresmac/.boot2docker/certs/boot2docker-vm/ca.pem
Writing /Users/foresmac/.boot2docker/certs/boot2docker-vm/cert.pem
Writing /Users/foresmac/.boot2docker/certs/boot2docker-vm/key.pem
Your environment variables are already set correctly.

(master) Violet:trunq-api > boot2docker version
Boot2Docker-cli version: v1.3.1
Git commit: 57ccdb8
```

**Comment by foresmac** [comment]
NM, I'm an idiot and didn't see that the stop, upgrade, start method only works with v1.4.1 or newer. Just re-instaleld from package and everything works so far.
